src/itest/resources/features/company-insolvency-delete.feature

```diff
@@ -1,13 +1,16 @@
 Feature: Delete company insolvency information
 
 
-  Scenario Outline: Delete company insolvency information successfully
+  Scenario: Delete company insolvency information successfully
 
     Given Insolvency data api service is running
-    And the insolvency information exists for "<companyNumber>"
-    When I send DELETE request with company number "<companyNumber>"
+    And the insolvency information exists for "CH3634545"
+    When I send DELETE request with company number "CH3634545"
     Then I should receive 200 status code
 
-    Examples:
-      | companyNumber |
-      | CH3634545     |
\ No newline at end of file
+  Scenario: Delete company insolvency information successfully
+
+    Given Insolvency data api service is running
+    And insolvency information is not added to database
+    When I send DELETE request with company number "CH3634545"
+    Then I should receive 404 status code
```

Comment:
Not sure why you have deleted existing steps, you could just send arbitrary value in the step 

When I send DELETE request with company number "Arbitrary Value"